common/beerocks/bwl/common/ap_wlan_hal.h

```diff
@@ -65,6 +65,41 @@ class ap_wlan_hal : public virtual base_wlan_hal {
 public:
     virtual ~ap_wlan_hal() = default;
 
+    /*!
+     * Enable the radio interface
+     *
+     * @return true on success or false on error.
+     */
+    virtual bool enable() = 0;
+
+    /*!
+     * Disable the radio interface
+     *
+     * @return true on success or false on error.
+     */
+    virtual bool disable() = 0;
+
+    /*!
+     * Set start_disabled flag
```

Comment:
This should have a better documentation. Based on this explanation, I wouldn't be able to implement the function and wouldn't know when to call it.

Reply:
It can be understood from the context.
There is a flag called "start_disabled" which this function is set on a specific vap_id. Also, it is possible to peek on the dwpal implementation to see how it implements it.

This is must be a new function since I don't recall having it in bwl, as well I didn't find any call to it on ap_manager/monitor.

@kantera800 can you add something about this function?